### DISCUSSION TITLE
Issues displaying images on the website

### DISCUSSION THREAD

**Discussion by apaditya7** [academicpages/academicpages.github.io#2286] [Q&A]
I am building a website and trying to display images onto the website through a markdown file. The website is being stored in github through jekyll. I am facing issues displaying images through the markdown file. Each time I try displaying the image it doesn't show up on the website. I am using a mac and vs code as my editor for reference.

I am referencing the image like this, 
#"![Alt text]( mobalytics.png )
<img width="227" alt="image" src="https://github.com/user-attachments/assets/67c5ceba-d9d8-4eca-a5e2-7beef24a6c2d">

 which is the exact relative path with regards to my markdown file. They are both in the same folder. Yet whenever I try displaying it I see no image on the screen. I've tried putting in the full path and still had no luck.

**Comment by rjzupkoii** [comment]
Which page are you adding the images to? Typically you need to use the HTML `img` tag with the path relative to the root of the repository, there are known issues with Markdown rendering differently between the template and GitHub (this is a known issue I'm looking into #2287).

**Comment by HaochongYANG** [comment]
you should try`![text](../your_image)`.  In my case `'/images/image.png'` doesn't work but `'../images/image.png'` works:)

**Comment by rlucas7** [comment]
https://github.com/academicpages/academicpages.github.io/issues/91#issuecomment-2427689676

^ my commend in the link fixed this issue for me
